clippy_lints/src/literal_representation.rs

```diff
@@ -131,7 +131,7 @@ pub fn format_numeric_literal(lit: &str, type_suffix: Option<&str>, float: bool)
 #[derive(Debug)]
 pub(super) struct NumericLiteral<'a> {
```

Comment:
Since this struct is now used in 3 modules, can you extract it in it's own `utils` module, please.

Reply:
Will do that.

Reply:
Done.